### DISCUSSION TITLE
Output multiple Videos and Video URLs with Magic Tags

### DISCUSSION THREAD

**Discussion by andrepetrini** [pods-framework/pods#6657] [Q&A]
Hi,
I'm having the same issue I had with images (https://github.com/pods-framework/pods/issues/5982), now I'm having with videos. I need the video URL, and in this case, also the placeholder poster to output it in a gallery with other images from the post. I'm using a Video field because I need it to be able to select multiple files.
I tried:
`{@videos}` outputs nothing.
`{@videos,pods_video}` if there is only one video, it outputs a full HTML tagged vídeo, but it won't work for me, as I need a specific markup for the gallery. BUT, if there is more than one video on the field, it doesn't work at all.
`{@videos._src}` outputs nothing.
Even though I'm not using an oembed file, I also tried `{@videos,wp_oembed_get}`, but got nothing from it.

Am I missing something, or this is behaving as expected?
Thank you

**Comment by JoryHogeveen** [comment]
Hi @andrepetrini 

`wp_oembed_get` will only work for external video's.
Video's work differently than images since these do not have a special magic tag like `@_img`.

You can fetch the video's like so:
```
[each videos]
{@_src,pods_video}
[/each]
```

Cheers, Jory

**Comment by JoryHogeveen** [comment]
Reopened for small enhancement:

Allow ID to be passed instead of src.
